package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@availity/favorites": "^1.1.9",
```

Comment:
Are we using availity? I don't see it imported anywhere. If not we should remove it from dependencies.